app/controllers/alerts_list_controller.rb

```diff
@@ -1,5 +1,6 @@
 class AlertsListController < ApplicationController
   extend ActiveSupport::Concern
+  include UiServiceMixin
```

Comment:
Aaah... missed that.. in that case, are you sure that's the right place?

Because .. that's not a list of all available icons, that's just a list of things that are shown in one of the topology views.

Any chance the UI can ask for a list of classes instead?